server/routes/viewingLicences/handlers/printLicence.ts

```diff
@@ -86,4 +107,24 @@ export default class PrintLicenceRoutes {
         </p>
      </span>`
   }
+
+  getConditionWithUpload = (additionalConditions: AdditionalCondition[]): number => {
+    let conditionId = 0
+    additionalConditions.forEach(condition => {
+      if (condition?.uploadSummary?.length > 0) {
+        conditionId = condition.id
+      }
+    })
+    return conditionId
+  }
+
+  getExclusionZoneDescription = (additionalConditions: AdditionalCondition[]): string => {
+    let description = ''
+    additionalConditions.forEach(condition => {
+      if (condition?.uploadSummary?.length > 0) {
+        description = condition.uploadSummary[0]?.description
+      }
+    })
+    return description
+  }
 }
```

Comment:
These both make assumptions that there is only one upload per condition, and at present, one upload for all conditions. Might need a rethink if we identify other uploads.